changelogs/gh-3680.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/core
+
+* Export lua_tonumberx and lua_tointegerx symbols (gh-3680).
```

Comment:
Should be in changelogs/unreleased/ and it would be better to add a short description into the file name.